datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/readmes.py

```diff
@@ -0,0 +1,71 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import re
+
+import click
+
+from ...utils import complete_valid_checks, get_valid_integrations, read_readme_file
+from ..console import CONTEXT_SETTINGS, abort, echo_failure, echo_info, echo_success
+
+IMAGE_EXTENSIONS = {"png", "jpg"}
+NON_TILE_INTEGRATIONS = {"sortdb", "hbase_master", "kube_proxy"}
```

Comment:
Should we fix those integrations readme(s) instead of using `NON_TILE_INTEGRATIONS` blacklist ?

Reply:
Should I just add empty sections?

Reply:
Created PRs to fix those integrations: https://github.com/DataDog/integrations-core/pull/7113 + https://github.com/DataDog/integrations-extras/pull/641